### DISCUSSION TITLE
Error: There is no node able to be processed.

### DISCUSSION THREAD

**Discussion by ANTIoffz** [alicevision/Meshroom#1697] [Q&A]
help(
![image](https://user-images.githubusercontent.com/103268389/170305910-c6d09a8c-ae03-4c21-8681-4802c4d40e91.png)

RuntimeError: [COMPUTATION] Impossible Process:
There is no node able to be processed.
WARNING:root:Retrying to obtain clipboard.

![image](https://user-images.githubusercontent.com/103268389/170306382-729111cc-44b9-49f1-9054-0fa8d90c8a8e.png)
Clear Pending Status not working

**Comment by natowi** [comment]
Save the project and recompute again. Open the node folder and delete the contents if necessary to reset the node
If this does not work try moving your images to another folder, this could be an permissions error.
